trunk/SUAVE/Methods/Weights/Correlations/UAV/empty.py

```diff
@@ -48,9 +48,11 @@ def empty(vehicle):
     
     
     # Airframe weight
-    Waf = (5.58*(S**1.59)*(AR**0.71))/g # All Samples
+    #Waf = (5.58*(S**1.59)*(AR**0.71))/g # 
     #Waf = (0.44*(S**1.55)*(AR**1.30))/g  # Top 19
     
+    Waf = 0.48*(S**1.5)*(AR**0.71)
```

Comment:
Where did this come from?

Reply:
Also maybe we should let the user decide which one they want to use.

Reply:
I might have copied over an old script. Issue Resolved